roles/2_setup-infra/files/set_default_kernel

```diff
@@ -0,0 +1,143 @@
+#!/bin/bash
+
+if [[ -z "$*" ]]; then
+    cat <<EOUSAGE
+    Usage: $(basename -- "$0") <kernel_version or 0>
+EOUSAGE
+    exit 1
+fi
+kernel_version="$1"
+now="$(date +"%Y-%m-%d_%H-%M-%S")"
+
+DISTRO_REGEX="CentOS|Red Hat Enterprise Linux Server|AlmaLinux|Debian|Ubuntu"
+
+function set_grub_default {
```

Comment:
btw, I know it says #!/bin/bash on the first line, but it is a good idea to always use foo(){} anyway:
http://mywiki.wooledge.org/BashPitfalls#function_foo.28.29